brave/browser/brave_browser_context.cc

```diff
@@ -91,6 +100,22 @@ BraveBrowserContext::BraveBrowserContext(const std::string& partition,
           base::Unretained(this)));
 #endif
   }
+
+  base::FilePath user_dir;
+  PathService::Get(brightray::DIR_USER_DATA, &user_dir);
+  CHECK(BrowserThread::CurrentlyOn(BrowserThread::UI));
+  web_database_ = new WebDatabaseService(user_dir.Append(kWebDataFilename),
```

Comment:
this path should have the partition name in it to avoid access from more than one browser context